### PR TITLE
SRVCOM-2932 redirects for release

### PR DESCRIPTION
**Pull request by briandooley**
Version(s):

Issue:
[SRVCOM-2932](https://issues.redhat.com/browse/SRVCOM-2932)

Link to docs preview:

N/A 

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by briandooley** [comment]
@gabriel-rh @kalexand-rh @gaurav-nelson @Srivaralakshmi @mramendi @aireilly Can one of you please give this a review? It does not require merging just yet.

File: .s2i/httpd-cfg/01-commercial.conf [hunk 1]
```diff
@@ -229,22 +229,22 @@ AddType text/vtt                            vtt
...
 
 
-    # redirect latest to 1.31
+    # redirect latest to 1.32
     RewriteRule ^serverless/?$ /serverless/latest [R=302]
-    RewriteRule ^serverless/latest/?(.*)$ /serverless/1\.31/$1 [NE,R=302]
+    RewriteRule ^serverless/latest/?(.*)$ /serverless/1\.32/$1 [NE,R=302]
 
     # redirect top-level without filespec to the about file
     RewriteRule ^serverless/(1\.28|1\.29|1\.30|1\.31)/?$ /serverless/$1/about/about-serverless.html [L,R=302]
```
Comment (Srivaralakshmi) [inline .s2i/httpd-cfg/01-commercial.conf:237]:
```suggestion
    RewriteRule ^serverless/(1\.28|1\.29|1\.30|1\.31|1\.32)/?$ /serverless/$1/about/about-serverless.html [L,R=302]
```
Missed this part.

**Review by Srivaralakshmi — COMMENTED** [review]
@briandooley Good work! Left a comment for your consideration. PTAL. Thanks!

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Pull request merged**